resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml

```diff
@@ -850,7 +850,7 @@ subjects:
   namespace: knative-eventing
 
 ---
-{{- if not (.Capabilities.APIVersions.Has "apiserversources.sources.knative.dev/v1alpha1") }}
+{{- if not (.Capabilities.APIVersions.Has "sources.knative.dev/v1alpha1/ApiServerSource") }}
```

Comment:
to make 1.13 to 1.14 work this not should be deleted

Reply:
And if crd should be on upgrade still part of release it should have labels "meta.helm.sh/release-name", release-namespace and managed-by

Reply:
upgrade kyma 1.13 to 1.14 should work but the only concern I have is upgrade from 1.14 to 1.14 this needs to be tested manually